dataclients/kubernetes/kube.go

```diff
@@ -788,6 +794,15 @@ func (c *Client) ingressToRoutes(state *clusterState) ([]*eskip.Route, error) {
 			}
 			annotationFilter += val
 		}
+
+		// add pre-configured default filters
+		if defFilter, ok := defaultFiltersOf(i, defaultFilters); ok {
+			if annotationFilter != "" {
+				annotationFilter += " -> "
+			}
+			annotationFilter += defFilter
```

Comment:
Do we really want to add this filters at the end?

We originally talked about putting monitoring at the incoming edge of the API gateway, i.e. at the begin of the filter chain.

Reply:
changed to prepend